aws/vpc/outputs.tf

```diff
@@ -0,0 +1,94 @@
+output "vpc_id" {
```

Comment:
Could we also output the vpc_name variable, for use in other modules? Or would we not typically output variables?

Reply:
We can output any of the Attributes of a resource. The list is in the registry, in the page of the resource itself, right at the end. Seems we can output the tags_all attribute, which in our case does contain the name of the vpc .. along with the rest of the tags as a map

Reply:
Great this sounds useful! I'll output the tags in the EC2 and ALB modules too